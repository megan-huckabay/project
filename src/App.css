.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
  text-align: center;
  
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }  
}

h1 {
  color: #000000;
  text-shadow: 4px 4px 15px rgba(240, 236, 243, 0.973);
  text-align: center;
  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
  font-size: 100px;
  margin: 5px 0 0; 
}

h6 {
  color: #000000;
  text-shadow: 4px 4px 15px rgba(125, 25, 187, 0.973);
  text-align: center;
  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
  font-size: 25px;
  margin: 5px 0 0; 
}
body {
  background-color: #eb9dbe;
  border: #282c34;
}