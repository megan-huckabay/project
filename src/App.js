import React from 'react';
import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
import './App.css';
import Home from './Components/Home';
import About from './Components/About';   
import Education from './Components/Education'; 
import Navigation from './Components/Navigation';  
import Experience from './Components/Experience';
import HireMe from './Components/HireMe';
import Contact from './Components/Contact';

function App() {
  return (
    <Router> 
      <header>
      <div className="app-container">
      <h1>Welcome to my Portfolio</h1>
      </div>
      </header>
      <div>
        <Navigation />

        <Routes>
          <Route path="/" element={<Home />} />
          <Route path="/About" element={<About />} />
          <Route path="/Education" element={<Education />} />
          <Route path="/Experience" element={<Experience />} />
          <Route path="/HireMe" element={<HireMe    />} />
          <Route path="/Contact" element={<Contact />} />
        </Routes>

        <Navigation />
      </div>
    </Router>
  );
}

export default App;
