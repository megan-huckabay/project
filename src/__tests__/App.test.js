// __tests__/App.test.js

import 'jest-styled-components';
import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

test('renders the App component', () => {
  const { getByText } = render(<App />);
  const welcomeText = getByText(/Welcome to my Portfolio/i);
  expect(welcomeText).toBeInTheDocument();
});

test('styles are applied correctly', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
