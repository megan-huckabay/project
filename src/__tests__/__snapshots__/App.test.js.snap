// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles are applied correctly 1`] = `
<div>
  <header>
    <div
      class="app-container"
    >
      <h1>
        Welcome to my Portfolio
      </h1>
    </div>
  </header>
  <div>
    <div
      class="navigation-container"
    >
      <nav>
        <ul>
          <li>
            <a
              class="nav-button"
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              class="nav-button"
              href="/About"
            >
              About
            </a>
          </li>
          <li>
            <a
              class="nav-button"
              href="/Education"
            >
              Education
            </a>
          </li>
          <li>
            <a
              class="nav-button"
              href="/Experience"
            >
              Experience
            </a>
          </li>
          <li>
            <a
              class="nav-button"
              href="/Contact"
            >
              Contact
            </a>
          </li>
        </ul>
      </nav>
    </div>
    <div
      class="home-container home-page"
    >
      <div
        class="home-section"
      >
        <h1>
          <strong>
            Hello there!
          </strong>
        </h1>
        <p>
          Ready to embark on a virtual journey into the depths of my digital domain?
          <br />
           I'm Megan Huckabay, your friendly guide
          <br />
           or just
          <strong>
             Meg
          </strong>
           if you're feeling casual.
          <br />
           Now that we've got the introductions sorted,
          <br />
           let's explore the intricacies of this React.js wonderland I've concocted.
          <br />
           Sprinkled at the summit and base of each page,
          <br />
           you'll discover buttons itching to whisk you away to sections that peel back the layers of my personality,
          <br />
           unveiling the real me – quirks, passions, and all.
          <br />
          <strong>
            Imagine this:
          </strong>
           placed perfectly at the top of my the education page,
          <br />
           a snapshot of my academic triumphs comes to life.
          <br />
           Feast your eyes on the 
          <strong>
            BloomTech Institute of Technology
          </strong>
           certificate,
          <br />
           standing tall beside my GCA score.
          <br />
           But that's not all – a carefully curated list of web development skills awaits,
          <br />
           ready to impress.
          <br />
           As you navigate through the tales of my past professional exploits,
          <br />
           brace yourself for a contact page that reads like a treasure map.
          <br />
           
          <strong>
            Email coordinates, LinkedIn links, resume routes, and GitHub trails
          </strong>
          <br />
           our key to unraveling the web of projects I've spun during my web development escapades.
          <br />
          So, fasten your seatbelt and enjoy the journey!
          <br />
           Whether you're keen on unraveling more mysteries or just in the mood for a good laugh,
          <br />
           feel free to drop me an email or connect on LinkedIn.
          <br />
          <br />
          <strong>
            Cheers!
          </strong>
        </p>
      </div>
    </div>
    <div
      class="navigation-container"
    >
      <nav>
        <ul>
          <li>
            <a
              class="nav-button"
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              class="nav-button"
              href="/About"
            >
              About
            </a>
          </li>
          <li>
            <a
              class="nav-button"
              href="/Education"
            >
              Education
            </a>
          </li>
          <li>
            <a
              class="nav-button"
              href="/Experience"
            >
              Experience
            </a>
          </li>
          <li>
            <a
              class="nav-button"
              href="/Contact"
            >
              Contact
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </div>
</div>
`;
